src/build.d

```diff
@@ -806,6 +853,18 @@ auto sourceFiles()
             statementsem.d staticassert.d staticcond.d strictvisitor.d target.d templateparamsem.d traits.d
             transitivevisitor.d typesem.d typinf.d utils.d visitor.d
         "),
+        /* TEMPORARY NOTES:
+            - Split for better encapsulation
+            - Added doc.h - Missing in win32.mak
+            - Removed objc.d - Already in fronted
+        */
+        frontendHeaders: fileArray(env["D"], "
+            aggregate.h aliasthis.h arraytypes.h attrib.h compiler.h complex_t.h cond.h
+            ctfe.h declaration.h dsymbol.h doc.h enum.h errors.h expression.h globals.h hdrgen.h
+            identifier.h id.h import.h init.h json.h module.h mtype.h nspace.h objc.h scope.h
+            statement.h staticassert.h target.h template.h tokens.h version.h visitor.h
+            " ~ (env["OS"] == "windows" ? "" : "libomf.d scanomf.d libmscoff.d scanmscoff.d")
+        ),
```

Comment:
Why do we need this? You can glob for all .d/.h files for the whitespace check.

Reply:
Glob matching is possible but i assumed that explicit sources would be preferred (based on #10446)

Reply:
I think for this check you simply want to check all files. It's only run on the CI anyhow. So imho there's no need to manually maintain this list.

Reply:
Done